static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -301,6 +302,11 @@ CStudioAuthoring.Module.requireModule(
               ? rteConfig.rteStylesheets.link
               : null;
 
+          rtePasteWordElements =
+            rteConfig.rtePasteWordElements && rteConfig.rtePasteWordElements.length != 0
+              ? rteConfig.rtePasteWordElements
+              : '';
```

Comment:
Looks like this will be a string. The first part of the condition is sufficient.
```suggestion
          rtePasteWordElements = rteConfig.rtePasteWordElements ? rteConfig.rtePasteWordElements : '';
```